### PR TITLE
Inventory system base updated req

### PR DESCRIPTION
**Pull request by SoundGuy**
extended the implemenation done by @trevorjmcdonald to include @ChemaDMK 's description.

### CODE REVIEW

**Comment by trevorjmcdonald** [comment]
Recommend merging this with PR# 100 rather than having two separate PRs.

**Comment by SoundGuy** [comment]
I sent a PR to your branch. once you do include them in your PR  i'll close this PR.

File: UOP1_Project/Assets/Scripts/Inventory/Item.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+{
+
+	// TODO: review if this enum redundant due to abstraction 
+	enum  ItemType
+	{
+		CharacterCustomisation,
+		Ingredients,
+		Utensils,
+		Dishes,
+	}
```
Comment (trevorjmcdonald) [inline UOP1_Project/Assets/Scripts/Inventory/Item.cs:15]: I agree with the comment, I think this is redundant - ideally we'll rely on inheritance to sort this out. For example, "Ingredient" already inherits Item.

**Comment by trevorjmcdonald** [comment]
> I sent a PR to your branch. once you do include them in your PR i'll close this PR.

I merged it into PR 100. If you have some time, go ahead and remove ItemType for now in a new PR and I'll merge that as well, just hit me up here or on the forum.

**Comment by SoundGuy** [comment]
closing this one.

**Pull request closed without merging**